src/garage/tf/algos/npo.py

```diff
@@ -68,6 +68,8 @@ class NPO(BatchPolopt):
         flatten_input (bool): Whether to flatten input along the observation
             dimension. If True, for example, an observation with shape (2, 4)
             will be flattened to 8.
+        fit_baseline_first (bool): Whether to fit baseline first or after. If
```

Comment:
don't forget to update the documentation of child classes

Reply:
Do you mean to make this NOT optional in all the super classes?

Reply:
Do you mean doing `fit_baseline_order=fit_baseline_order` in constructor of child classes of `NPO`, like `PPO`?